content/Certification/certifying_a_product/ota_recert.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "OTA and Re-certification"
+chapter = false
+weight = 11
++++
+
+![](../imgs/ota_recert.png)
+
+Placeholder with questions:
+- ANN - do you have text around why OTAs and recertification are important
+- PETR - does the compatibility matrix cover which ecosystems pick up OTAs from the DCL vs. internal?
```

Comment:
uh...no idea what Petr's github is, but imagine I tagged him here.

Reply:
I don't believe that it does. And the compatibility matrix is still an internal-only doc for now.